run-build.sh

```diff
@@ -59,6 +73,11 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+# $args array may have empty elements in it.
+# The easiest way to remove them is to cast to string and back to array.
+temp="${args[@]}"
+args=($temp)
```

Comment:
Yes. This breaks the case above. However, we haven't run into any issues regarding this yet. So, I would say, let's keep it.